guide/using-python.md

```diff
@@ -92,3 +92,54 @@ Open the DOS console (CMD.EXE) and type:
 PYTHON_PATH\Scripts\pip.exe install opencv-python
 ```
 where `PYTHON_PATH` is the path to the Python installation directory, for example `C:\Python27`.
+
+
+### Use an alternative Python version
+
+As described above, the Python libraries for Webots are precompiled with Python 2.7.
+It is possible to use another Python version such as Python 3 or macOS brew Python,
+by recompiling these libraries.
+This documentation is generic and may not cover all the cases.
+This task should be reserved for users that have strong experience in their OS,
+build systems and terminal management.
```

Comment:
That sounds a bit odd... I would reformulate it simply this way:

"This task requires advanced knowledge in operating systems management."